packages/osl-management-console-image/generated/modules/osl-management-console/configure-httpd.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+set -e
+
+SOURCES_DIR=/tmp/artifacts
+SCRIPT_DIR=$(dirname "${0}")
+MGMT_CONSOLE_HOME="${KOGITO_HOME}/management-console"
+
+# Configure the default httpd conf
+echo "Mutex posixsem" >> "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i -e "/#ServerName www.example.com:80/aHeader set Content-Security-Policy \"frame-ancestors 'self';\"" "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i -e 's/Options Indexes FollowSymLinks/Options -Indexes +FollowSymLinks/' "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i "s/Listen 80/Listen ${SONATAFLOW_MANAGEMENT_CONSOLE_PORT}/g" "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i "s/#ServerName www.example.com:80/ServerName 127.0.0.1:${SONATAFLOW_MANAGEMENT_CONSOLE_PORT}/g" "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i '$ a ServerTokens Prod' "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i '$ a ServerSignature Off' "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+sed -i -e '/<Directory "\/var\/www\/html">/a    RewriteEngine on\n    RewriteCond %{REQUEST_FILENAME} -f [OR]\n    RewriteCond %{REQUEST_FILENAME} -d\n    RewriteRule ^ - [L]\n    RewriteRule ^ index.html [L]' "${HTTPD_MAIN_CONF_PATH}/httpd.conf"
+
+
+# Set the required paths
+mkdir -p "${MGMT_CONSOLE_HOME}/launch"
+
+# Copy the entrypoint and other init scripts
+cp -v "${SCRIPT_DIR}"/added/* "${MGMT_CONSOLE_HOME}"/launch
+
+# Fixing permissions
+chmod +x "${MGMT_CONSOLE_HOME}/launch/entrypoint.sh" "${MGMT_CONSOLE_HOME}/image-env-to-json-standalone"
+chown -R "${USER_ID}" "${MGMT_CONSOLE_HOME}"
+
```

Comment:
I think the changes introduced as part of PR https://github.com/kiegroup/kie-tools/pull/53/ are missing here. @fantonangeli @ricardozanini IIUC, this file is created in `generated` folder based on upstream configure-httpd.sh, so I suppose the fix for PR https://github.com/kiegroup/kie-tools/pull/53/ was not sent upstream or maybe not required?

Reply:
It must be upstream too. @fantonangeli can u do upstream and once we sync we get the fix here too.

Reply:
Pull request opened: https://github.com/apache/incubator-kie-tools/pull/2719